catalog/app/containers/Bucket/PackageUpdateDialog.js

```diff
@@ -402,7 +401,7 @@ function FilesInput({ input, meta, uploads, setUploads, errors = {} }) {
               !error && warn && classes.dropMsgWarn,
             )}
           >
-            {label}
+            <M.Typography variant="body2">{label}</M.Typography>
```

Comment:
couldnt we just use `span`?

Reply:
Yes. Do you prefer `<span>` here?

Reply:
well, the change would be smaller, and it also seems a little simpler